README.md

```diff
@@ -117,6 +117,18 @@ Only works with version '2' docker-compose.yml configuration files
 
 This option can also be configured on the agent machine using the environment variable `BUILDKITE_PLUGIN_DOCKER_COMPOSE_IMAGE_REPOSITORY`.
 
+## `image-tag` (optional)
+
+If you want to use a custom tag, please set this option.
+
+The default is `"$project_name-$COMPOSE_SERVICE_NAME-build-$BUILDKITE_BUILD_NUMBER"`.
```

Comment:
@lox any further thoughts on this? We're quite keen to see it adopted rather than continuing the monkey patch on our side :)